src/tsserver.d.ts

```diff
@@ -0,0 +1,12 @@
+import * as ts from 'typescript';
```

Comment:
I'm surprised that these typings don't already exist somewhere in the TypeScript project. Do we have to use these custom ambient types?